docs/containers-registries.conf.5.md

```diff
@@ -254,6 +259,28 @@ in order, and use the first one that exists.
 
 Note that a mirror is associated only with the current `[[registry]]` TOML table. If using the example above, pulling the image `registry.com/image:latest` will hence only reach out to `mirror.registry.com`, and the mirrors associated with `example.com/foo` will not be considered.
 
+### Enabling Additional Layer Store to access to private registries
+
+The `additional-layer-store-auth-helper` option enables to pass registry
+credentials to Additional Layer Store so that it can access private registries.
+
+Additional Layer Store needs to provide a helper binary. This helper binary is
+registered to `additional-layer-store-auth-helper` option. Every time pulling of
+an image happens, the specified helper binary is executed and receives registry
```

Comment:
“Every time an image is read using the `docker://` transport”, or something like that:
- c/image doesn’t know about pulling
- `skopeo inspect docker://…` is not a pull, but it _would_ send credentials

Also in the `additional-layer-store-auth-helper` option documentation above.

Reply:
Thanks for the comments. Fixed.